vowpalwabbit/core/include/vw/core/gen_cs_example.h

```diff
@@ -12,6 +12,12 @@ namespace VW
 {
 namespace details
 {
+class cb_to_cs_per_model_state
```

Comment:
Might be a good opportunity to move this and name is something more appropriate. Since it is only relevant for MTR

Reply:
Agree with the name change. I think moving cb_to_cs makes sense for a separate PR. @lalo thoughts on naming?

Reply:
no idea but yeah could be renamed based on that other pr of interleaved